qualang_tools/digital_filters/filters.py

```diff
@@ -81,9 +89,9 @@ def high_pass_exponential(x, t):
     return np.exp(-x / t)
 
 
-def single_exponential_correction(A: float, tau: float, Ts: float = 1):
+def single_exponential_correction(A: float, tau: float, Ts: float = 1, qop_version: Enum = QOP_VERSION.QOP222):
```

Comment:
I would keep the default to all functions (besides the `calc` one above) at `qop_version: None`.
If we do want to keep this as is, then I would add another constant above:
`LATEST_QOP_VERSION =  QOP_VERSION.QOP222` and use it everywhere.

Reply:
My preference goes towards setting the latest QOP version as the default one, otherwise people will not specify it and it may lead to very bad results. For instance if the feedback tap is not capped and it is set to 1, then the OPX will output garbage data from -0.5V to +0.5V which can kill some spin qubit devices (it almost happened to me during an installation). I like your LATEST_QOP_VERSION  idea!